Fixed OSPP profile title and description in RHEL7

mpreisler opened the pull request:
The OSPP profile title said USGCB which was confusing in the GUI tools and HTML guides.

shawndwells commented:
@mpreisler - This will be the RHEL7 USGCB and STIG submission, so naming as US Gov Baseline seems appropriate. Many don't know who/what NIAP is, hence the simplified "US Gov Baseline" name. How are the tools getting confused? Is it name consistency between RHEL6/RHEL7 profiles?

mpreisler commented:
Yes, inconsistency between RHEL6 and 7.

I didn't know OSPP was USGCB and STIG submission so I figured it could be a copy-paste error.

iankko commented:
@shawndwells 
> @mpreisler - This will be the RHEL7 USGCB and STIG submission, so naming as US Gov Baseline seems appropriate. Many don't know who/what NIAP is, hence the simplified "US Gov Baseline" name. 

So maybe just change it to  "United States Government Security Baseline" (e.g. drop the "Configuration" keyword? / replace with "Security") to avoid the inconsistency?

iankko commented:
@shawndwells 
So should we close this PR as not necessary for now? Can't decide if it should be merged or not based on the discussion above (the particular code change should be safe to be merged, but that's not the issue here).

Thanks, Jan.

mpreisler commented:
@shawndwells @iankko I have changed the commit. Now it has USGCB title, similar to what RHEL6 content has. I have changed the description according to feedback received here.

iankko commented:
@shawndwells 
> @shawndwells @iankko I have changed the commit. Now it has USGCB title, similar to what RHEL6 content has. I have changed the description according to feedback received here.

Shawn, once got to it, could you have a look & let me know if you are OK with the actual form? (If Jenkins build succeeds, I would merge this doing the review just on my own).

Thanks, Jan.

shawndwells commented:
In the RHEL6 days, USGCB was derived from seemingly arbitrary NIST 800-53 controls. For RHEL7, we're deriving both the DoD STIG and NIST USGCB from the OSPPv4 requirements. It makes sense the RHEL6 and 7 names will be different, as the requirements differ slightly and RHEL7 reflects consolidation of USGCB and STIG into a single baseline.

However customers have "brand loyalty" to USGCB and STIG. Meaning, generally, a DoD/IC customer will read the term "USGCB" and believe the baseline is irrelevant because they follow the STIG.

Because of this, to remove contention from naming, I suggest we either:
(a) Name the profile "United States Government Configuration Baseline" as this is a neutral term. Yes, it's the long form of USGCB.... but most people didn't know what USGCB stood for anyway.

(b) Name the profile "United States Government Configuration Baseline (USGCB / STIG)" ... I like this best because it reflects STIG / USGCB consolidation and includes the keywords that both civilian and DoD/IC customers will look for

iankko commented:
Thanks, Shawn.

> (b) Name the profile "United States Government Configuration Baseline (USGCB / STIG)" ... I like this best because it reflects STIG / USGCB consolidation and includes the keywords that both civilian and DoD/IC customers will look for

If I should choose between the two (a) and b)) I like b) option more too.

shawndwells commented:
On 10/6/15 9:57 AM, iankko wrote:
>
> Thanks, Shawn.
>
>     (b) Name the profile "United States Government Configuration
>     Baseline (USGCB / STIG)" ... I like this best because it reflects
>     STIG / USGCB consolidation and includes the keywords that both
>     civilian and DoD/IC customers will look for
>
> If I should choose between the two (a) and b)) I like b) option more too.
>

More suggestions certainly welcome! Those 2 options were all I could 
think of.

mpreisler commented:
I went with option b) and push-forced the change.

iankko commented:
LGTM (IMHO I would replace 'RHEL7' with the official name in the profile description, but that can be fixed in future versions).

ACK.
